src/qt/proposalview.cpp

```diff
@@ -70,14 +70,6 @@ ProposalList::ProposalList(const PlatformStyle *platformStyle, QWidget *parent)
 {
     ui->setupUi(this);
 
-    if (platformStyle->getUseExtraSpacing()) {
-        ui->hlayout->setSpacing(5);
-        ui->hlayout->addSpacing(26);
-    } else {
-        ui->hlayout->setSpacing(0);
-        ui->hlayout->addSpacing(23);
-    }
-
     if (platformStyle->getUseExtraSpacing()) {
```

Comment:
this breaks visibility on macOS

Reply:
@HashUnlimited I made a change. Can you please test again?